README.md

```diff
@@ -98,7 +98,8 @@ In your application, e.g.
         .withDockerHost("tcp://my-docker-host.tld:2376")
         .withDockerTlsVerify(true)
         .withDockerCertPath("/home/user/.docker/certs")
-        .withDockerConfig("/home/user/.docker")
+        .withDockerConfig("/home/user/.docker/config.json")
+        //.withDockerConfig("/home/user/.docker/.dockercfg")   //Your docker auth config file
```

Comment:
please place here comment/description what is the difference. I.e. since XXX docker cli version